src/hooks/useRefresh.ts

```diff
@@ -42,6 +42,25 @@ export async function refreshRegistries(): Promise<void> {
   await clearServiceCache();
 }
 
+const throttledRefreshRegistries = throttle(
+  async () => {
+    try {
+      await throttledRefreshRegistries();
```

Comment:
This should be calling the other function, right?
```suggestion
      await refreshRegistries();
```